futures-util/src/lib.rs

```diff
@@ -9,6 +9,7 @@
 extern crate futures_core;
 extern crate futures_io;
 extern crate futures_sink;
+extern crate either;
```

Comment:
`#[cfg(feature = "std")]`?

Reply:
Shouldn't be needed; I'm already carefully dispatching to `either`'s own `std` feature.